doc/source/whatsnew/v0.21.0.txt

```diff
@@ -830,6 +830,7 @@ Other API Changes
 - Restricted DateOffset keyword arguments.  Previously, ``DateOffset`` subclasses allowed arbitrary keyword arguments which could lead to unexpected behavior.  Now, only valid arguments will be accepted. (:issue:`17176`).
 - Pandas no longer registers matplotlib converters on import. The converters
   will be registered and used when the first plot is draw (:issue:`17710`)
+- ``NaT`` division with :class:`datetime.timedelta` will now return NaN instead of raising (:issue:`17876`)
```

Comment:
Double-tick the NaN.

Reply:
Also, what is the rationale for returning `NaN` ?  Why not return `NaT` instead?

Reply:
It also occurs to me that you *might* not need a `whatsnew` given that this is internal.

Reply:
> Double-tick the NaN. 

Just pushed a fix.

> Also, what is the rationale for returning `NaN` ? Why not return `NaT` instead?

This is just taking behavior that currently applies to `Timedelta` and applying the same to `timedelta`.

Reply:
Hmm...seems a little non-intuitive to me, but then again, I'm not going to push changing the norm without getting feedback from other maintainers:

@jreback @jorisvandenbossche ?

Reply:
I think the idea is that in this context `NaT` is ducking as a `Timedelta`, so `Y = NaT / Timedelta(X)` is solving for `Timedelta(X) * Y = NaT` --> `Y = np.nan`

Reply:
Ah, I see.  Right, that makes sense.

Reply:
Yes, that seems to make sense to me, similar as:

```
In [5]: np.nan / 5
Out[5]: nan
```

And also this is how `pd.NaT / pd.Timedelta(..)` is already working? In that case I would rather call it a bug fix.

Reply:
> And also this is how pd.NaT / pd.Timedelta(..) is already working? 

Yes.